pyproject.toml

```diff
@@ -20,11 +24,6 @@ classifiers= [
     "Operating System :: OS Independent",
 ]
 dynamic = ["version", "readme", "dependencies"]
-dependencies = {file = ["requirements.txt"]}
```

Comment:
I agree that was probably the build problem.

Reply:
No problem at all. That happens.

And it shows again that PSpaMM should get a CI. :)